docs/architecture.md

```diff
@@ -2,34 +2,40 @@
 
 Very brief notes for now.
 
-## Backend/GUI
+## Backend and GUI
 
-At the highest level, Anki is logically separated into two parts.
+At the highest level, the Anki codebase is logically separated into two parts:
 
-A neat visualization of the file layout is available here:
-<https://mango-dune-07a8b7110.1.azurestaticapps.net/?repo=ankitects%2Fanki>
-(or go to <https://githubnext.com/projects/repo-visualization#explore-for-yourself> and enter `ankitects/anki`).
+- the GUI code
+- the backend code
 
-### Library (rslib & pylib)
+### GUI (`qt/` and `ts/`)
 
-The Python library (pylib) exports "backend" methods - opening collections,
-fetching and answering cards, and so on. It is used by Anki’s GUI, and can also
-be included in command line programs to access Anki decks without the GUI.
+Anki’s GUI is written parts in Python using the toolkit Qt (via Qt's Python bindings known as PyQt), and parts in a mix of TypeScript, HTML, and CSS.
 
-The library is accessible in Python with "import anki". Its code lives in
-the `pylib/anki/` folder.
+The Python code that implements Anki’s GUI (using PyQt) is located in `qt/aqt/`. It’s made available as a Python package named `aqt` (installable with `pip install 'aqt[qt6]'`) and is accessible through `import aqt` in Python code.
 
-These days, the majority of backend logic lives in a Rust library (rslib, located in `rslib/`). Calls to pylib proxy requests to rslib, and return the results.
+The web code is split between `ts/` and `qt/aqt/data/web/`.
+The majority of new code now goes into `ts/` rather than `qt/aqt/data/web/` and, during the build process, gets copied into `qt/aqt/data/web/` instead.
```

Comment:
```suggestion
The majority of new code now goes into `ts/` rather than `qt/aqt/data/web/`.
```

The old text was incorrect - it's copied to a different location, and not really relevant.